syft/workers/websocket_client.py

```diff
@@ -51,6 +55,7 @@ def search(self, *query):
         return sy.serde.deserialize(response)
 
     def _send_msg(self, message: bin, location) -> bin:
+        self.ws.settimeout(TIMEOUT_INTERVAL)
```

Comment:
This should not be here (if you see the message below this method should never be called)

Reply:
You are right. It should be removed: i just added it to address any possible call to websockets.